code/modules/projectiles/guns/ballistic.dm

```diff
@@ -440,6 +440,7 @@
 
 /obj/item/gun/ballistic/attack_self(mob/living/user)
 	if(HAS_TRAIT(user, TRAIT_GUNFLIP))
+		src.SpinAnimation(7,1)
```

Comment:
dont do src.proc

Reply:
I'm not aware of the policy on this, what should I do instead.

Reply:
```suggestion
		SpinAnimation(7,1)
```

Reply:
like so

Reply:
Thanks